### PR TITLE
[1LP][RFR] Adding BZ wrapper for test_individual_host_creds

### PR DESCRIPTION
**Pull request by rhrazdil**
Purpose or Intent
=================

__Wraping__ test_individual_test_creds test cases due to created BZ's.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 36e7189a38fb9ebc6f211784b2e872ec8e8a985e:

Lint OK! :cake: :punch: :cookie: .

* 36e7189a38fb9ebc6f211784b2e872ec8e8a985e
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 17406a2c398c715794a8e0d9b6d7c422631b7408:

Lint OK! :cake: :punch: :cookie: .

* 36e7189a38fb9ebc6f211784b2e872ec8e8a985e
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f05ec7d0c0171b39ff2042715453d7a72473c8de:

Lint OK! :cake: :punch: :cookie: .

* f05ec7d0c0171b39ff2042715453d7a72473c8de
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

File: cfme/tests/infrastructure/test_individual_host_creds.py [hunk 1]
```diff
@@ -33,6 +33,9 @@ def get_host_data_by_name(provider_key, host_name):
 
 
 # Tests to automate BZ 1278904
+@pytest.mark.meta(blockers=[BZ(1516849,
+                            forced_streams=['5.8', '5.9', 'upstream'],
+                            unblock=lambda provider: provider.type != 'rhevm')])
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_individual_host_creds.py]: **Required**

Do not directly compare type strings, use the provider classes and the `provider.one_of()` method - lots of examples throughout the code base.

**Pull request merged**